idea_town/experiments/models.py

```diff
@@ -60,6 +61,18 @@ class UserInstallation(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
+    def save(self, *args, **kwargs):
+        if self.pk is None:
+            self.experiment.installation_count += 1
```

Comment:
I'm not sure this approach will be thread-safe. Maybe use an F expression instead?

 `self.experiment.installation_count = F('installation_count') + 1`

Docs: https://docs.djangoproject.com/en/1.9/ref/models/expressions/#avoiding-race-conditions-using-f